tumblr/tumblr.php

```diff
@@ -184,10 +187,11 @@ function tumblr_jot_nets(&$a,&$b) {
 }
 
 
-function tumblr_settings(&$a,&$s) {
+function tumblr_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.